snowflake/src/integration/java/org/apache/iceberg/snowflake/NamespaceTests.java

```diff
@@ -0,0 +1,268 @@
+    } finally {
+      dropSchemaIfExists(schema);
+    }
+  }
+
+  @ParameterizedTest
+  @ValueSource(
+      strings = {
+        "\"Schema123\"",
+        "\"MySchema\"",
+        "\"my.schema\"",
+        "\"my schema\"",
+        "\"My 'Schema'\"",
+        "\"3rd_schema\"",
+        "\"$Ischema\"",
+        "\"H@!!.0-w*r()^'\"",
+        "\"идентификатор\""
+      })
+  public void testLoadNamespaceWithQuotedIdentifier(String schema)
+      throws SQLException, InterruptedException {
+    String dbName = TestConfigurations.getInstance().getDatabase();
+    try {
+      clientPool.run(conn -> conn.createStatement().execute("use " + dbName));
+      createOrReplaceSchema(schema);
+      Assertions.assertThatNoException()
+          .isThrownBy(
+              () -> {
+                snowflakeCatalog.loadNamespaceMetadata(Namespace.of(dbName, schema));
+              });
+    } finally {
+      dropSchemaIfExists(schema);
+    }
+  }
+
+  @Test
+  public void testDropNamespaceNotSupported() throws SQLException, InterruptedException {
+    String schema = "Schema123";
+    String dbName = TestConfigurations.getInstance().getDatabase();
+    try {
+      clientPool.run(conn -> conn.createStatement().execute("use " + dbName));
+      createOrReplaceSchema(schema);
+      Assertions.assertThatThrownBy(
+              () -> {
+                snowflakeCatalog.dropNamespace(Namespace.of(dbName, schema));
+              })
+          .isInstanceOf(UnsupportedOperationException.class)
+          .hasMessageContaining("SnowflakeCatalog does not currently support dropNamespace");
+    } finally {
+      dropSchemaIfExists(schema);
+    }
+  }
+
+  @Test
+  public void testSetPropertiesNotSupported() throws SQLException, InterruptedException {
+    String schema = "Schema123";
+    String dbName = TestConfigurations.getInstance().getDatabase();
+    try {
+      clientPool.run(conn -> conn.createStatement().execute("use " + dbName));
+      createOrReplaceSchema(schema);
+      Assertions.assertThatThrownBy(
+              () -> {
+                snowflakeCatalog.setProperties(
+                    Namespace.of(dbName, schema), new HashMap<String, String>());
+              })
+          .isInstanceOf(UnsupportedOperationException.class)
+          .hasMessageContaining("SnowflakeCatalog does not currently support setProperties");
+    } finally {
+      dropSchemaIfExists(schema);
+    }
+  }
+
+  @Test
+  public void testRemovePropertiesNotSupported() throws SQLException, InterruptedException {
+    String schema = "Schema123";
+    String dbName = TestConfigurations.getInstance().getDatabase();
+    try {
+      clientPool.run(conn -> conn.createStatement().execute("use " + dbName));
+      createOrReplaceSchema(schema);
+      Assertions.assertThatThrownBy(
+              () -> {
+                snowflakeCatalog.removeProperties(
+                    Namespace.of(dbName, schema), new HashSet<String>());
```

Comment:
nit (I think checkstyle should even complain about this): `Sets.newHashSet()` or `ImmutableSet.of()`